scripts/check_frontend_coverage.py

```diff
@@ -39,7 +39,7 @@
     'admin-misc-tab.directive.ts',
     'admin-navbar.directive.ts',
     'admin-page.directive.ts',
-    'admin-prod-mode-activities-tab.directive.ts',
```

Comment:
I don't see any changes in test files, then why are we removing these filenames? Can you please explain? (Sorry for so many questions, your answer will be very helpful for me to understand the reason behind the changes.)

Reply:
@DubeySandeep, these components don't have any code that can be tested, so their coverage is reported as 100% when they are migrated to angular. So I think @krishita30j would have gotten an error while pushing that they have to remove the following from the list. BTW, these files don't a spec file.